index.html

```diff
@@ -0,0 +1,135 @@
+            </p>
+            <nav class="nav">
+                <i class="fa-solid fa-bars"></i>
+            </nav>
+        </div>
+    </header>
+    <section>
+    <div class="presentation">
+            <h1> Hey There, I'm Ghislain </h1>
+            <h2>I'm a software developer</h2>
+        <p class="text">I can help you build a product, feature or website Look through some of my Work and experience ! If you like what you see and you need coded,don't hestiate to contact me.</p>
+            <ul class="icons">
+                <li><a href=""><i class="fa-brands fa-twitter"></i></a></li>
+                <li><a href=""><i class="fa-brands fa-linkedin-in"></i></a></li>
+                <li><a href=""><i class="fa-brands fa-github"></i></a></li>
+                <li><a href=""><i class="fa-solid fa-m"></i></a></li>
+                <li><a href=""><i class="fa-brands fa-angellist"></i></a></li>
+            </ul>
+    </div>
+<div class="works">
+    <p class="more">
+        <i class="fa-solid fa-angle-down"></i>
+    </p>
+    <h3 class="works-title">
+        My Recenter Works
+    </h3>
+    <div class="columns">
+        <div class="card">
+            <div class="works-heading"></div>
+            <div class="works-footer">
+                <p class="Project-title">Multi-Post stories Gain+Glory</p>
+                    <ul class="list-inline">
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>Javascript</li>
+                        <li>html</li>
+                    </ul>
+                 <input type="submit" value="See Project" class="btn"/>
+            </div>
+        </div>
+        <div class="card">
+            <div class="works-heading"></div>
+            <div class="works-footer">
+                <p class="Project-title">Multi-Post stories Gain+Glory</p>
+                    <ul class="list-inline">
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>Javascript</li>
+                        <li>html</li>
+                    </ul>
+                <input type="submit" value="See Project" class="btn"/>
+            </div>
+        </div>
+        <div class="card">
+            <div class="works-heading"></div>
+            <div class="works-footer">
+                <p class="Project-title">Multi-Post stories Gain+Glory</p>
+                    <ul class="list-inline">
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>Javascript</li>
+                        <li>html</li>
+                    </ul>
+                 <input type="submit" value="See Project" class="btn"/>
+            </div>
+        </div>
+
+        <div class="card">
+            <div class="works-heading"></div>
+            <div class="works-footer">
+                <p class="Project-title">Multi-Post stories Gain+Glory</p>
```

Comment:
- [ ] I would suggest you replace the `p` tag with `hx` tag as this is the title, not paragraph